pkg/logger/fields.go

```diff
@@ -20,10 +22,20 @@ func Any(key string, val interface{}) Field {
 }
 
 func Err(err error) Field {
+	if os.Getenv("SENTRY_DSN") != "" {
+		sentry.CaptureException(err)
+	}
+
 	return zap.Error(err)
 }
 
 func Errs(msg string, errs []error) Field {
+	if os.Getenv("SENTRY_DSN") != "" {
+		for _, err := range errs {
+			sentry.CaptureException(err)
+		}
+	}
+
```

Comment:
this should be added to globals.go -> func Error

these functions are for defining fields in the log message

Reply:
They are put here so I can get the raw `Error` object - if I put them in `func Error` I only get access to the zap-encoded field type

Reply:
The Error function takes these as arguments